frontend/components/buttons/DropdownButton/_styles.scss

```diff
@@ -19,7 +19,7 @@
     padding: 0;
     margin: 0;
     display: none;
-    z-index: 2;
```

Comment:
this worries me a bit as its app wide so lets just keep an eye on if there are any bugs with the dropdown button that come up